### PR TITLE
feature/git

### PR DESCRIPTION
**Pull request by xavigs89**

### CODE REVIEW

File: staff/xavi-gonzalez/git/README.md [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+  feature/playground
+  feature/space-invaders
+  feature/strings
+  main
+```
+
+## git branch -D
+delete a branch
+```sh
+$ git branch feature/branch name to remove
```
Comment (Barrufat) [inline staff/xavi-gonzalez/git/README.md]: Check this command, it's not correct yet🤯

File: staff/xavi-gonzalez/git/README.md [hunk 2]
```diff
@@ -0,0 +1,146 @@
...
+```
+
+## git commit -m
+prepares file to be uploaded with an specific written command
+```sh
+$ git commit -m 'add git #162'
+```
+
+## git push
+upload new commit
```
Comment (Barrufat) [inline staff/xavi-gonzalez/git/README.md:97]: The command implementation should appear inside a back ticks section, just as the other command examples 🙀

File: staff/xavi-gonzalez/git/README.md (hunk 1 above)
Comment (xavigs89) [reply staff/xavi-gonzalez/git/README.md]: ok done. You are the best teacher, for real

File: staff/xavi-gonzalez/git/README.md (hunk 2 above)
Comment (xavigs89) [reply staff/xavi-gonzalez/git/README.md:97]: ok done. Again, you are the best teacher

File: staff/xavi-gonzalez/git/README.md (hunk 1 above)
Comment (Barrufat) [reply staff/xavi-gonzalez/git/README.md]: Good job! You are the best student xD haha

**Pull request merged**